external/storm-kafka-client/pom.xml

```diff
@@ -52,7 +52,6 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
-            <version>${storm.kafka.client.version}</version>
```

Comment:
What's behind of the change? We're maintaining both `storm.kafka.version` and `storm.kafka.client.version`, and this line is for picking correct version.